src/pages/Rockets.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import RocketItem from '../components/rockets/elementRockets';
+import styles from '../components/rockets/Rockets.module.css';
 
 function Rockets() {
+  const rocketList = useSelector((state) => state.rocketReducer);
   return (
-    <div>
-      Rockets
-    </div>
+    <ul className={styles.flex}>
+      {rocketList.map((item) => (
+        <RocketItem
+          desc={item.desc}
+          key={item.id}
+          id={item.id}
+          name={item.name}
+          image={item.flickr_images}
```

Comment:
Nice rendering list of rockets (as per design) using image of first rocket image in the array of flickr_images. 🎉